beta-src/src/components/ui/WDPhaseUI.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { useState, useEffect } from "react";
+import { Box } from "@mui/material";
+import { gameOverview } from "../../state/game/game-api-slice";
+import { useAppSelector } from "../../state/hooks";
+import Season from "../../enums/Season";
+import UIState from "../../enums/UIState";
+import WDCountdownPill from "./WDCountdownPill";
+import WDGamePhaseIcon from "../svgr-components/WDGamePhaseIcon";
+import WDPillScroller from "./WDPillScroller";
+
+const WDPhaseUI: React.FC = function (): React.ReactElement {
+  const [currentSeason, setCurrentSeason] = useState<Season>(Season.SPRING);
+  const [currentYear, setCurrentYear] = useState<number>(1901);
+  const [timerDisplayState, setTimerDisplayState] = useState(true);
+
+  const { phaseMinutes, season, year } = useAppSelector(gameOverview);
+
+  const phaseTime = phaseMinutes * 60;
+  const endTime = Math.floor(Date.now() / 1000) + phaseMinutes * 60;
```

Comment:
we should move this logic out as a helper function. phaseMinutes calculation should only be done once. here, we are doing it on line 19 and 20.

Reply:
The pill timer component requires both a phaseTime and endTime prop. 
I moved the end time logic out to a helper function but kept the phaseTime variable!